src/feature.js

```diff
@@ -46,14 +43,25 @@ class FxABrowserFeature {
       });
     }
 
+    // For control, the add-on is still installed but removed
+    // from the browser toolbar. This will allow us to end the
+    // study and have the user fill out survey.
+    if (studyInfo.variation.name === "control") {
```

Comment:
Moving this at the bottom and removing the return allows us to send telemetry when in the control variation, naming sign-in, sign-out events.

Reply:
Should the calls to `listen`, `addListener`, `setTitle` and `setIcon` be moved into an `else` branch of this condition? That all feels like work that doesn't need to take place for the control group.

Reply:
(and `updateState` too I guess)

Reply:
On a similar note, I wonder if we should make the call to `sendTelemetry` the very last thing this function does, after all the UI updates. (on the basis that everything which has any kind of visual effect should be painted as soon as possible then whatever's left can happen after)

Reply:
We don't need to set title and icon for control, but I think will will still need to have the listeners setup. After speaking with @irrationalagent, tracking sync events for sign-in/sign-out for both variations would make the analysis a little easier.